.github/workflows/pre-commit.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v2
         with:
-          python-version: "3.10"
+          python-version: "3.11"
```

Comment:
```suggestion
          cache: pip
          cache-dependency-path: ./pyproject.toml
          python-version: "3.11"
```

Reply:
Here, it doesn't make sense (as it only installs `pre-commit` and the `pyproject.toml` is not the right cache key.

Reply:
But I've added `cache: pip`

Reply:
Hmm, doesn't seem to work: `No file in /home/runner/work/capella-dockerimages/capella-dockerimages matched to [**/requirements.txt], make sure you have checked out the target repository`. Removed it again.